Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2.py

```diff
@@ -1831,19 +2072,333 @@ def export_pcap_file_command(client: Client, args: Dict) -> List:
+                                                      interface_id=interface_id,
+                                                      firewall_policy=firewall_policy,
+                                                      firewall_port_policy=firewall_port_policy,
+                                                      ips_policy=ips_policy,
+                                                      custom_policy_json_key=custom_policy_json_key,
+                                                      custom_policy_json_value=custum_policy_json_value
+                                                      )
+
+    return CommandResults(
+        readable_output='Policy assigned successfully.',
+        raw_response=response
+    )
+
+
+def list_interface_policy_command(client: Client, args: Dict) -> CommandResults:
+    """
+
+    Args:
+        client (Client):  - A McAfeeNSM client.
+        args (Dict): - The function arguments.
+
+    Returns:
+        A CommandResult object with a list of policies.
+    """
+    domain_id = arg_to_number(args.get('domain_id'))
+    interface_id = arg_to_number(args.get('interface_id'))
+    limit = arg_to_number(args.get('limit', 50))
+    all_results = argToBoolean(args.get('all_results', False))
+
+    response = client.list_interface_policy_request(domain_id=domain_id, interface_id=interface_id)
+    if not interface_id:
+        all_policies = (
+            response.get('policyAssignmentsList')
+            if all_results
+            else response.get('policyAssignmentsList', {})[:limit]
+        )
+    elif all_results:
+        all_policies = response
+    else:
+        all_policies = [response][:limit]
+    # Capitalize the keys of the policies list and keeping the rest of the keys as is.
+    capitalize_policies = []
+    for policy in all_policies:
+        policy = {k[:1].upper() + k[1:]: v for k, v in policy.items()}
+        capitalize_policies.append(policy)
+
+    return CommandResults(
+        readable_output=tableToMarkdown(
+            name='Interface policy List', t=capitalize_policies, removeNull=True
+        ),
+        outputs_prefix='NSM.InterfacePolicy',
+        outputs=capitalize_policies,
+        raw_response=response
+    )
+
+
+def get_device_configuration_command(client: Client, args: Dict) -> CommandResults:
+    """
+
+    Args:
+        client (Client):  - A McAfeeNSM client.
+        args (Dict): - The function arguments.
+
+    Returns:
+        A CommandResult object with the device configuration information.
+    """
+    device_id = arg_to_number(args.get('device_id'))
+
+    response = client.get_device_configuration_request(device_id=device_id)
+    # Capitalize the keys of the policies list and keeping the rest of the keys as is.
+    capitlize_response: Dict[str, Any] = {k[:1].upper() + k[1:]: v for k, v in response.items()}
+    # build a new dict with the keys and values of the nested dict
+    iner_dict: Any = capitlize_response.get('PendingChanges')
```

Comment:
I believe you meant - **inner** dict. 
```suggestion
    inner_dict: Any = capitlize_response.get('PendingChanges')
```